jenkinsfiles/dev

```diff
@@ -182,6 +182,41 @@ pipeline {
                 }
             }
         }
+
+        stage('Update IM Play dev instance') {
+            if (env.GIT_BRANCH == 'master') {
+                steps {
+                    script {
+                        withCredentials([usernamePassword(credentialsId: 'e2e-im-user', passwordVariable: 'PASSWORD', usernameVariable: 'USER_EMAIL')]) {
+                            dir('im-db-manager') {
+                                sparseCheckout('https://github.com/dhis2-sre/im-database-manager', 'master', '/scripts')
+
+                                dir('/scripts') {
+                                    env.DATABASE_ID = sh(
+                                            returnStdout: true,
+                                            // TODO: How do we avoid hard coding the name/version?
+                                            script: "./list.sh | jq -r '.[] | select(.Name == \"whoami\") | .Databases[] | select(.Name == \"Sierra Leone - 2.39.0.sql.gz\") | .ID'"
```

Comment:
Name of the database should be something like “Sierra Leone - dev.sql.gz” (based on [the db manager seed script](https://github.com/dhis2-sre/im-database-manager/blob/master/scripts/seed.sh)); there’re no databases currently uploaded on prod, though.

Reply:
"Sierra Leone - 2.39.0.sql.gz" is the name of the database on the manager.